e2e/index.ts

```diff
@@ -0,0 +1,12 @@
+import { exampleUrlTokenFromFilePath } from './e2eApi'
+
+const e2eExampleContext = require.context('./tests/', false, /-example.tsx$/)
+
+export const Routes = e2eExampleContext.keys().reduce((acc, key) => {
```

Comment:
```suggestion
export const routes = e2eExampleContext.keys().reduce((acc, key) => {
```

nit: it's just variable, so it can be camelcased